shongo-controller/src/main/java/cz/cesnet/shongo/controller/booking/request/AbstractReservationRequestAuxData.java

```diff
@@ -0,0 +1,30 @@
+package cz.cesnet.shongo.controller.booking.request;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import cz.cesnet.shongo.controller.booking.specification.Specification;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.Immutable;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@Entity
+@Immutable
```

Comment:
@Immutable a @Setter nad jedným POJO nedáva zmysel. 
Najvačší zmysel mi dáva pridávať k všetkým @Entity 
@Data (namiesto @Getter a @Setter)
@NoArgsConstructor (potrebný pre Hibernate)

Reply:
`@Immutable` sa pouziva, pretoze je to entita k `view`u. Je ale pravda, ze `@Setter` nedava zmysel.